lib/travis/build/script/julia.rb

```diff
@@ -57,7 +57,7 @@ def setup
             when 'windows'
               sh.cmd %Q{curl -A "$CURL_USER_AGENT" -s -L --retry 7 -o julia-installer.exe '#{julia_url}'}
               sh.cmd 'chmod +x julia-installer.exe'
-              sh.cmd %Q{cmd.exe /c 'julia-installer.exe /S /D=C:/julia'}
+              sh.cmd %Q{pwsh.exe -c 'Start-Process -FilePath julia-installer.exe -ArgumentList "/D=C:/julia" -NoNewWindow -Wait'}
```

Comment:
`sh.cmd %Q{pwsh.exe -c 'Start-Process -FilePath julia-installer.exe -ArgumentList "/S /D=C:/julia" -NoNewWindow -Wait'}`

It looks like both `/S` and `-NoNewWindow` are needed.